ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/objectstore/StoreManager.java

```diff
@@ -145,20 +145,20 @@ private static final ObjectStoreAPI initStore(String name)
     {
         ObjectStoreEnvironmentBean storeEnvBean = BeanPopulator.getNamedInstance(ObjectStoreEnvironmentBean.class, name);
         String storeType = storeEnvBean.getObjectStoreType();
-        ObjectStoreAPI store;
 
         try
         {
-            store = ClassloadingUtility.loadAndInstantiateClass(ObjectStoreAPI.class, storeType, name);
+            ObjectStoreAPI store = ClassloadingUtility.loadAndInstantiateClass(ObjectStoreAPI.class, storeType, name);
+
+            store.start();
```

Comment:
store can still be null so I think you will have the same issue.

Why is the load failing? It sounds like it ought to be a fatal error if the load and instantiate fails.